smart_card_connector_app/src/chrome-api-provider.js

```diff
@@ -445,6 +445,7 @@ GSC.ConnectorApp.ChromeApiProvider = class extends goog.Disposable {
     this.subscribeToChromeApiEvents_();
 
     this.serverRequester_.addOnDisposeCallback(() => this.dispose());
+    goog.log.info(logger, 'ChromeApiProvider was intialized successfully.');
```

Comment:
This log is emitted in addition to `Creating ChromeApiProvider <...>`. Seems excessive; maybe leave only one of the two?

Reply:
done